src/Akka.Streams.Kafka/Stages/Consumers/Abstract/BaseSingleSourceLogic.cs

```diff
@@ -85,6 +108,47 @@ public override void PostStop()
         /// </summary>
         protected abstract void ConfigureSubscription();
 
+        protected void ConfigureSubscription(Action<IImmutableSet<TopicPartition>> partitionsAssignedCb,
```

Comment:
overload of `ConfigureSubscription` that automatically sets up all `ISubscriptionRequest`s with the `KafkaConsumerActor`.

Worth noting that callback handlers et al are all now set through the `Subscribe` / etc messages and are not passed into the `KafkaConsumerActor`'s CTOR anymore.